File: POO_course/PHP/uberVan.php
<?php
require_once('Car.php');
class UberVan extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model) {
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }

    public function setPassenger($passenger) {
        if($passenger == 6) {
            $this->passenger = $passenger;
        } else {
            echo "You must assign 6 passsengers"."<br>";
        }
    }
}
?>

File: POO_course/Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    /* public UberVan(String license, Account driver, 
    Map<String, ArrayList<String,Integer>> typeCarAccepted, ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    } */

    public UberVan(String license, Account driver) {
        super(license, driver);
    }

    public void setPassenger(Integer passenger) {
        if(passenger == 6) {
            this.passenger = passenger;
        } else {
            System.out.println("You must assign 6 passengers");
        }
    }
}
